angr/sim_type.py

```diff
@@ -1079,19 +1097,43 @@ def parse_type(defn, preprocess=True):
     if pycparser is None:
         raise ImportError("Please install pycparser in order to parse C definitions")
 
-    defn = 'typedef ' + defn.strip('; \n\t\r') + ' QQQQ;'
+    defn = re.sub(r"/\*.*?\*/", r"", defn)
 
-    if preprocess:
-        defn = do_preprocess(defn)
+    parser = pycparser.CParser()
+
+    # pycparser hack to parse type expressions
+    errorlog = logging.getLogger('yacc logger')
+    errorlog.setLevel(logging.ERROR)
```

Comment:
users generally consider it bad when there are loggers they can't control the level of. I would stick the declaration and setlevel of errorlog in the top scope, and probably change the name to `__name__ + ".yacc"`

Reply:
I have declared the errorlog and have set its level in the top scope.